adr/0002-API-params-naming.md

```diff
@@ -0,0 +1,27 @@
+# 1. Record architecture decisions
```

Comment:
Should this be `2. Name API parameters to match DPS APIs` or similar?

Reply:
good point.